tests/model/DefaultModelControllerTests.cpp

```diff
@@ -59,28 +59,28 @@ TEST_F(ModelControllerTests, AddTask_ShouldAskModelToAddTaskAndReturnSuccessWith
     EXPECT_CALL(*task_validator_, ValidateTask(expected_task_)).WillOnce(Return(expected_task_validator_result));
     EXPECT_CALL(*model_, AddTask(expected_task_)).WillOnce(Return(expected_model_result));
 
-    ModelController controller { std::move(model_), std::move(task_validator_), std::move(persistence_factory_) };
+    DefaultModelController controller { std::move(model_), std::move(task_validator_), std::move(persistence_factory_) };
```

Comment:
Consider wrapping this in `unique_ptr` and placing in `SetUp`.

Reply:
will be done in refactoring PR.